web/src/app/review/page.tsx

```diff
@@ -196,7 +202,7 @@ export default function Review() {
                         <CardFooter className="flex flex-row justify-center gap-4">
                             {activeCard.overdue && (
                                 <p className="text-destructive text-sm">
-                                    {`Due ${daysSince(new Date(activeCard.next_review_date))} days ago`}
+                                    {`Due ${daysSince(new Date(activeCard.next_review_date))} day(s) ago`}
```

Comment:
Could we add the "s" if it's > 1? Maybe assign the value from daysSince for the review date to a constant. Might be some neat syntax out there for that, not sure.

Reply:
Will fix in a future PR 👍